packages/core/application/core-application-browser-internal/src/application_service.tsx

```diff
@@ -345,9 +346,15 @@ export class ApplicationService {
             setAppLeaveHandler={this.setAppLeaveHandler}
             setAppActionMenu={this.setAppActionMenu}
             setIsMounting={(isMounting) => httpLoadingCount$.next(isMounting ? 1 : 0)}
+            customLogo$={
+              this.customLogo$ ? this.customLogo$.pipe(takeUntil(this.stop$)) : undefined
+            }
```

Comment:
Optional: I think having this prop always be mandatory and passing down an empty observable would simplify the underlying checks ( `customLogo={this.customLogo$ ? this.customLogo$.pipe(takeUntil(this.stop$)) : EMPTY}`